pkg/server/server.go

```diff
@@ -384,6 +386,12 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgav1.ListObjectsRequ
 		methodName,
 	).Observe(queryCount)
 
+	requestDurationByQueryHistogram.WithLabelValues(
+		openfgav1.OpenFGAService_ServiceDesc.ServiceName,
+		methodName,
```

Comment:
We really should try to follow the naming standards of the gRPC service definition. Looking at `openfgav1.OpenFGAService_ServiceDesc.Methods`, the names are of the form `Check`, `ListObjects`, etc..

Reply:
openfgav1.OpenFGAService_ServiceDesc.Methods returns array of [MethodDesc](https://pkg.go.dev/google.golang.org/grpc@v1.59.0#MethodDesc), which I don't is what we want.  As for whether "ListObjects" vs. "listobjects", I believe that was brought into by [comment](https://github.com/openfga/openfga/pull/959#discussion_r1310781858) made in the [PR 959](https://github.com/openfga/openfga/pull/959).